templates/quickstart-confluence-master.template.yaml

```diff
@@ -955,52 +989,52 @@ Mappings:
       Jvmheap: 12288m
   AWSRegionArch2AMI:
     ap-northeast-1:
-      HVM64: ami-0a46812db6fcb9888
+      HVM64: ami-0aacdff09302c167c
```

Comment:
I think we should undo the ami changes and not pass down the synchrony properties. It will reduce the amount of risk introduced by this change and we believe it is not necessary anyway. Changing the default value for synchrony DB config in the quickstart will create variance amongst default values for customers which is not ideal.